app/MMB/src/scenes/timeline/events/AirportArrivalTimelineEvent.js

```diff
@@ -0,0 +1,49 @@
+// @flow
+
+import React from 'react';
+import { Text, StyleSheet } from '@kiwicom/mobile-shared';
+import { Translation, DateFormatter } from '@kiwicom/mobile-localization';
+import idx from 'idx';
+
+import TimelineEvent from '../TimelineEvent';
+import Icon from '../TimelineEventIcon';
+import DateLocation from '../TimelineEventDateLocation';
+
+type Props = {|
+  +data: ?Object,
```

Comment:
This doesn't look right - you should return Relay fragment container from this component and fetch what's necessary here (and use generated Flow type for this component)...